### ISSUE TITLE
C-g doesn't abort `add-require`

### ISSUE THREAD

**Issue by Malabarba** [clojure-emacs/clj-refactor.el#296]
The same is probably true of other commands. If I invoke `ar`, start typing a namespace name, and then change my mind, `C-g` doesn't behave as expected.

I would expect it to either (1) go back to what I had before the command started or (2) abort the command and leave the ns form as it is now.

Instead, when I hit `C-g`, it leaves the `ns` form as it is now and then continues with the command (which means that it cleans up and evaluates the ns form). Evaluating the ns form is particularly problematic because I'm guaranteed to have an unfinished namespace name in there.

**Comment by magnars** [comment]
Good point. This has annoyed me as well. Maybe we should ditch our yasnippet dependency and add a simpler placeholder minor-mode.

**Comment by MalloZup** [comment]
autogenerated with https://github.com/MalloZup/doghub:  issue inactive since 450 days. Please update the issue or close it

**Comment by expez** [comment]
@bbatsov I don't think we need the stalebot in this project.  

I don't see any value in closing issues just because they're old.  They're still relevant and if someone wants to help out they now have a list of issues to take on.

**Comment by MalloZup** [comment]
@expez  HI! I just run the bot in this project org. The bot is deactivate and was only 1 shot.

I apoligize for the noise and feel free to remove the comment . i agree with you there. :+1:

**Comment by expez** [comment]
@MalloZup Wait, you just took it upon yourself to update every issue with a staleness report?  If none of the other maintainers asked you to do this, could you please have the bot clean up after itself and delete all the comments it made?  Thanks!

**Comment by MalloZup** [comment]
@expez  I should have removed the issues and comment.
